gradoop-store/gradoop-hbase/src/main/java/org/gradoop/storage/impl/hbase/constants/HBaseConstants.java

```diff
@@ -95,8 +77,4 @@
    * Default cache size for scans in HBase.
    */
   public static final int HBASE_DEFAULT_SCAN_CACHE_SIZE = 500;
```

Comment:
Using EPGMGraphOutput::DEFAULT_CACHE_SIZE instead, remove this

Reply:
done.